src/zigbee_stack.rs

```diff
@@ -91,13 +104,42 @@ pub enum NwkRouteStatus {
 pub struct NwkRoutingTableEntry {
     pub destination: Nwk,
     pub status: NwkRouteStatus,
+
+    // A flag indicating that the destination indicated by this address does not store
```

Comment:
Just a tip for the future: these kinds of comments are useful enough they should be doc comments instead. As doc comments (`/// doc comment text`) they show up in hover docs (useful when reading code) and generated documentation.

(Sorry if this is a bit out of nowhere I was browsing through the recent PR's to get up to speed when I saw this)

Reply:
Thanks! I'll switch to the triple backslash syntax when I rewrite this section, I'm currently doing a second pass of the routing implementation.